### PR TITLE
I saw this while searching for something else and it triggered me

### PR DESCRIPTION
**Pull request by Amelia0010**

### CODE REVIEW

**Comment by Hopekz** [comment]
hvEnMpKl

**Comment by jcatjmeow** [comment]
Make sure to add if (TRUE) in front of the whole thing so the program doesn't get confused

File: code/modules/antagonists/cult/cult_items.dm [hunk 1]
```diff
@@ -976,9 +976,7 @@ GLOBAL_VAR_INIT(curselimit, 0)
 		to_chat(holder, "<span class='cult italic'>The shield's illusions are back at full strength!</span>")
 
 /obj/item/shield/mirror/IsReflect()
-	if(prob(block_chance))
-		return TRUE
-	return FALSE
+	return prob(block_chance)
```
Comment (TheGamerdk) [inline code/modules/antagonists/cult/cult_items.dm]: return FALSE

Comment (TheGamerdk) [inline code/modules/antagonists/cult/cult_items.dm]: Replace with
`if(prob(block_chance))
		return TRUE	
	return FALSE`

File: code/modules/antagonists/cult/cult_items.dm [hunk 2]
```diff
@@ -976,9 +976,7 @@ GLOBAL_VAR_INIT(curselimit, 0)
 		to_chat(holder, "<span class='cult italic'>The shield's illusions are back at full strength!</span>")
 
 /obj/item/shield/mirror/IsReflect()
-	if(prob(block_chance))
```
Comment (TheGamerdk) [inline code/modules/antagonists/cult/cult_items.dm]: Duplicate of #42

**Pull request merged**